src/rust/grapl-service/src/serialization/zstd_proto_graph.rs

```diff
@@ -58,7 +58,93 @@ impl CompletionEventSerializer for SubgraphSerializer {
             return Ok(vec![]);
         }
 
-        // TODO: This should be a monitor
+        for edge_list in subgraph.edges.values_mut() {
+            edge_list.edges.sort_unstable();
+            edge_list.edges.dedup();
+        }
+
+        // TODO: This should be a metric
```

Comment:
Should this TODO be implemented in this PR?

Reply:
I don't think so, it wasn't introduced in this PR